doc_admin/AdminManual.adoc

```diff
@@ -105,17 +111,17 @@ TIP: If when running Open Hospital the application appears small-scaled or blurr
 
 Depending on the operating system, there are several software tools to install and manage a MySQL server. In most cases just a standard installation package of the software is needed, paying attention to configuration options during the installation process.
 
-For instance, on a Microsoft Windows® system, MySQL Workbench® serves as a client, while on Linux based systems the MySQL Administrator and MySQL Query Browser tools are used to perform the same tasks. Some newer distributions (distros) include MySQL Workbench.
+For instance, on a Microsoft Windows® system, MySQL Workbench® can be used as a client to connect to the database server and perform the needed tasks, while on Linux based systems the MySQL Administrator and MySQL Query Browser tools are used to perform the same tasks. Some newer distributions (distros) include MySQL Workbench.
 
-The best approach for installation is to use the command-line MySQL client which uses essentially the same commands and syntax for every platform.
+The suggested approach for installation is to use the command-line MySQL client which uses essentially the same commands and syntax for every platform.
 
 NOTE: See Software Compatibility Matrix in order to identify the correct MariaDB / MySQL version.
 
 NOTE: MariaDB is the preferred alternative for the DBMS.
 
 ===== 3.1.3.1 MySQL - Server installation
 
-Please referer to the latest MySQL Server online documentation for downloadfootnote:[https://www.mysql.com/downloads/] and installationfootnote:[https://dev.mysql.com/doc/] information while using this document.
+Please referer to the latest MySQL Server online documentation for downloadfootnote:[https://mariadb.org/download/] and installationfootnote:[https://mariadb.org/documentation/] information while using this document.
```

Comment:
should `the latest MySQL Server` be `the latest MariaDB Server` given the URL given?

Reply:
fixed